chpl/chpl-api/e2e/collections/certified-product-controller.postman_collection.json

```diff
@@ -1,54 +1,922 @@
+									}
+								},
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"/{certifiedProductId} end point should return status 200 for valid API Key\", function () {\r",
+											"    pm.response.to.have.status(200);\r",
+											"});\r",
+											"const expresult = pm.response.json()\r",
+											"pm.test(\"/{certifiedProductId} end point response should have all RWT fields\", function () {\r",
+											"    pm.expect(expresult).to.have.property('rwtPlansUrl');\r",
+											"    pm.expect(expresult).to.have.property('rwtPlansCheckDate');\r",
+											"    pm.expect(expresult).to.have.property('rwtResultsUrl');\r",
+											"    pm.expect(expresult).to.have.property('rwtResultsCheckDate');\r",
+											"    pm.expect(expresult).to.have.property('rwtEligibilityYear');\r",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "API-Key",
+										"value": "{{apiKey}}",
+										"type": "text"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "{{url}}/rest/certified_products/10295",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"rest",
+										"certified_products",
+										"10295"
+									]
+								},
+								"description": "GET /{certifiedProductId} end point should return status 200 and response should have all RWT fields for valid API Key"
+							},
+							"response": []
+						},
+						{
+							"name": "GET /{certifiedProductId}- Invalid API Key",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"/{certifiedProductId} end point should return status 401 for Invalid API Key\", function () {\r",
+											"    pm.response.to.have.status(401);\r",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "API-Key",
+										"value": "1290y7y8h8ko90b8ecd0596c98ae9094",
+										"type": "text"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"url": {
+									"raw": "{{url}}/rest/certified_products/10295",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"rest",
+										"certified_products",
+										"10295"
+									]
+								},
+								"description": "GET /{certifiedProductId} end point should return status 401 for Invalid API Key"
+							},
+							"response": []
+						},
+						{
+							"name": "GET /{certifiedProductId} - without API Key",
```

Comment:
I'm not sure it's of much use to run all of these tests without an API key, or with an invalid one. I could see doing it one time, maybe, but this looks like a lot of tests that aren't testing something that we think could ever really fail. I'd much rather have some data driven tests where we have a few listings that we _know_ have some measures, and some that we _know_ don't, and then expect to get those values in the responses

Reply:
removed those tests